frontend/packages/core-auth/src/components/AuthWrapper.tsx

```diff
@@ -157,7 +183,7 @@ const AuthWrapper: React.FC<AuthWrapperProps> = ({
           customLoginComponent({ login: handleLogin })
         ) : (
           <button type="button" onClick={handleLogin}>
-            Login
+            Login {tokenResponse?.accessToken}
```

Comment:
We probably don't want this

Reply:
oh yeah sorry, that was just for testing something